src/style.css

```diff
@@ -2,0 +1,37 @@
+html {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  margin: 0;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../src/images/star-icon.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child){
+  margin: 0 4px 4px 0;
+}
```

Comment:
Be consistent with your margins. It's recommended to add margins only to the top or bottom of elements to prevent potential margin collapse issues. In this case, you're adding margins to the right and bottom, which could lead to inconsistencies when stacking elements or when they wrap.